linera-service/tests/end_to_end_tests.rs

```diff
@@ -141,11 +141,16 @@ impl Application {
         serde_json::from_value(response_body["accountInfo"]["orders"].clone()).unwrap()
     }
 
-    async fn add_liquidity(&self, owner: AccountOwner, token0_amount: u64, token1_amount: u64) {
+    async fn add_liquidity(
```

Comment:
Methods specific to one particular application should be named accordingly:

```suggestion
    async fn amm_add_liquidity(
```

Or (maybe even cleaner?) we could create a wrapper type `struct AmmApp(Application)` and _that_ could have an `add_liquidity` method.

Reply:
Ah yeah, that's similar to what I was doing before. I can split the methods again in a cleaner way